man/systemd-cryptenroll.xml

```diff
@@ -251,55 +256,65 @@
           <!-- See: https://github.com/tianocore-docs/edk2-TrustedBootChain/blob/main/4_Other_Trusted_Boot_Chains.md -->
           <!-- See: https://wiki.archlinux.org/title/Trusted_Platform_Module#Accessing_PCR_registers -->
 
-          <tgroup cols='2' align='left' colsep='1' rowsep='1'>
+          <tgroup cols='3' align='left' colsep='1' rowsep='1'>
             <colspec colname="pcr" />
+            <colspec colname="string_alias" />
             <colspec colname="definition" />
 
             <thead>
               <row>
                 <entry>PCR</entry>
+                <entry>alias</entry>
                 <entry>Explanation</entry>
               </row>
             </thead>
 
             <tbody>
               <row>
                 <entry>0</entry>
+                 <entry>core-firmware</entry>
                 <entry>Core system firmware executable code; changes on firmware updates</entry>
               </row>
 
               <row>
                 <entry>1</entry>
+                <entry>platform-config</entry>
                 <entry>Core system firmware data/host platform configuration; typically contains serial and model numbers, changes on basic hardware/CPU/RAM replacements</entry>
               </row>
 
               <row>
                 <entry>2</entry>
+                <entry>external-code</entry>
                 <entry>Extended or pluggable executable code; includes option ROMs on pluggable hardware</entry>
               </row>
 
               <row>
                 <entry>3</entry>
+                <entry>external-firmware</entry>
                 <entry>Extended or pluggable firmware data; includes information about pluggable hardware</entry>
               </row>
 
               <row>
                 <entry>4</entry>
+                <entry>boot-loader</entry>
                 <entry>Boot loader and additional drivers; changes on boot loader updates. The shim project will measure the PE binary it chain loads into this PCR. If the Linux kernel is invoked as UEFI PE binary, it is measured here, too. <citerefentry><refentrytitle>sd-stub</refentrytitle><manvolnum>7</manvolnum></citerefentry> measures system extension images read from the ESP here too (see <citerefentry><refentrytitle>systemd-sysext</refentrytitle><manvolnum>8</manvolnum></citerefentry>).</entry>
               </row>
 
               <row>
                 <entry>5</entry>
+                <entry>boot-loader-config</entry>
                 <entry>GPT/Partition table; changes when the partitions are added, modified or removed</entry>
               </row>
 
               <row>
                 <entry>6</entry>
+                <entry>power-events</entry>
                 <entry>Power state events; changes on system suspend/sleep</entry>
```

Comment:
Again, the spec says PCR6 is "Host Platform Manufacturer Specific Measurements". I don't see anything about power events being measured into this. So this is just wrong or a big assumption/generalization.

Reply:
> Again, the spec says PCR6 is "Host Platform Manufacturer Specific Measurements". I don't see anything about power events being measured into this. So this is just wrong or a big assumption/generalization.

Thank you so much sir.
It's super exciting, that top experts here are taking their time to comment on this PR. You guys make me feel great.
Thank you sir. 
Let me quickly mention everyone else so we have a consensus.

I appreciate the time you take to even look at this PR, I appreciate even more, the fact that you're guiding.
Thank you sir.

Reply:
We've been going back and forth on this. I proposed "power-events" because various wikis say that "Resume from S4 and S5 Power State Events" is recorded there. This was actually true in the past: "TCG PC Client Implementation for Conventional BIOS 1.2 revision 1.00" says "Entities to be Measured:
1. Resuming from S5 or S4". But "TCG PC Client Platform Firmware Profile Specification revision 1.05" says "Previously defined measurements for PCR[6] from the PC Client Implementation Specification for Conventional
BIOS have been deprecated". So let's assume that we know nothing about this PCR. Let's go back to the "host-platform" name then.

Reply:
I think we should just not assign a name to this PCR. Simply drop it from the table in docs and in the enum.

Reply:
> I think we should just not assign a name to this PCR. Simply drop it from the table in docs and in the enum.

My thoughts too at some point. I was like sine PR[8] is missing anyways, we could just leave out PCR[6] if it causes so much back and forth.